### DISCUSSION TITLE
wfreedrp settings configuration when on slow network

### DISCUSSION THREAD

**Discussion by kaizennet** [FreeRDP/FreeRDP#8773] [Q&A]
Hello,  I'm using wfreerdp from command line which works great when at home.  Grateful to have it as I often encountered Credential Manager: UI Host issues through native Windows RDP and never have any issue with wfreerdp. 

We live kinda far out, when working outside the home on a slow (molasses) network / mobile hotspot inside an rdp window each click can be painfully slow.    Is there any one (or more) setting that might be recommended to change / disable when working on very slow connection.    

My initial thought was to change the color depth to 16bpp but noticed in the case "_What is the best format and bpp for performance in headless client? #8211_" user **akallabeth** mentioned  "_Performance differences in RDP sessions are negligible, so best always use 32bpp_"    

Is there anything other possibility at the expense of appearance / extra functionality?   I typically use fullscreen w float bar.  Any performance difference fullscreen vs windowed?  Grasping at straws ;)   

Thanks again for the excellent utility.

~ GP

**Comment by akallabeth** [comment]
@kaizennet the best option is not to use any settings at all, then the client and server adjust according to available bandwidth. the answer in #8211 did not have much to do with network data (`gdi_init` just initializes the local framebuffer that receives the decoded image data).
(default with FreeRDP since a few versions now is `/network:auto` which enables exactly that)
